Tests/XcodeGenKitTests/ProjectGeneratorTests.swift

```diff
@@ -110,7 +110,7 @@ func projectGeneratorTests() {
             }
 
             $0.it("generates configs") {
-                let spec = ProjectSpec(
+                let spec = ProjectSpec.Project(
                     basePath: "",
```

Comment:
It seems everything in Tests has a different renaming applied. Does leaving off the `ProjectSpec` module cause naming conflicts?

Reply:
Hmm - we shouldn't have a naming conflict, will update to match the rest of the renames!